GVRf/Framework/framework/src/main/jni/objects/textures/base_texture.h

```diff
@@ -98,41 +71,18 @@ class BaseTexture: public Texture {
     virtual void runPendingGL() {
```

Comment:
Old getId assumes being called on the GL thread. If we want to hide the threading from developers it has to change somehow.

runPendingGL I changed because it was not being actually used as intended in this repo. In this repo the things these code handles are already handled on the Java side, see update, updateCall. Not sure if any is better but this was easier to remove.